call_times.py

```diff
@@ -27,7 +27,7 @@ def overlap(time, callTimes):
             max_agents = max_n
         time += 1
     
-    print('Runtime:', (tt.time() - start_time)*1000)
+    //print('Runtime:', (tt.time() - start_time)*1000)
```

Comment:
Comments in Python starts with `#` instead of `//`

Reply:
Sorry, just played around in C recently 😅